src/HeaderCell.tsx

```diff
@@ -138,15 +157,18 @@ export default function HeaderCell<R, SR>({
     }
   }
 
-  function onClick() {
+  function onClick(event: React.MouseEvent<HTMLSpanElement>) {
+    if (ignoreClickRef.current) return;
```

Comment:
Maybe we should just have a real DOM element with `stopPropagation()` on it, would simplify the JS a little bit.

Reply:
Yeah. let's do it